docs/documentation/pack-docs.md

```diff
@@ -86,8 +86,31 @@ An example of including a YouTube video can be seen at the [Zoom Feed README](ht
 ## Pack Keywords, Tags, Use-cases & Categories
 In order to classify packs and make them easier to find, there are a number of pack metadata elements that you can use:
 - ***Use-case*** -- a name of a situation in which packs can be used - usually the pack name. The optional approved values for use-cases are:
```

Comment:
I'm not sure how the current list is done - but if you check the preview, there are 2 lists now here and in tags